### PR TITLE
[wg/das] 2023 draft updated per comments at the strategy issue

### PR DESCRIPTION
**Pull request by himorin**
as title. for comments provided to the strategy issue

/cc @anssiko @reillyeon / @plehegar @npdoty @svgeesus

### CODE REVIEW

**Review by anssiko — APPROVED** [review]

File: 2023/das-wg-charter.html [hunk 1]
```diff
@@ -180,15 +180,14 @@ <h2 id="scope">
...
           </li>
-          <li>monitoring the ambient light level of the device’s environment
+          <li>responding to ambient light levels of the device’s environment
           </li>
-          <li>monitoring changes in motion and orientation of the hosting
+          <li>reacting to changes in motion or orientation of the hosting
           device
           </li>
-          <li>obtaining and monitoring the location of the hosting device
+          <li>providing location information, and not necessarily location information of the hosting device
```
Comment (npdoty) [inline 2023/das-wg-charter.html]: This section seems significantly better to me. The phrasing here could be slightly different if you want, "providing location information, either of the hosting device or of a location of interest"

**Review by npdoty — APPROVED** [review]

**Review by svgeesus — APPROVED** [review]
Looking good

File: 2023/das-wg-charter.html (hunk 1 above)
Comment (himorin) [reply 2023/das-wg-charter.html]: I would want to hear from chairs on this point, but Anssi marked with thumb-up, so I count that provided alternative is better.
(although, personally, alternative one is much better than what I've replaced for clarification, of course...)

**Comment by himorin** [comment]
any other comment??

**Comment by himorin** [comment]
see no further comment/suggestion over one day. merging this.
thank all for review+

File: 2023/das-wg-charter.html [hunk 2]
```diff
@@ -1607,6 +1604,22 @@ <h3 id="history">
             </tr>
           </tbody>
         </table>
+        </section>
+
+        <section id="changelog">
+          <h3>Change log</h3>
+
+          <!-- Use this section for changes _after_ the charter was approved by the Director. -->
```
Comment (astearns) [inline 2023/das-wg-charter.html:1612]: don’t need 'by the Director' here

**Pull request merged**